docs/demonstration/README.md

```diff
@@ -3,8 +3,14 @@
 These notebooks demonstrate how to install and run the extractors and the
 Learning Management System Data Store Loader (LMS-DS Loader) in a Jupyter
 notebook. The same code can be copied over into a stand-alone Python script
-instead of running inside the notebook. Each notebook has its own
-getting-started instructions.
+instead of running inside the notebook.
+
+Before running the notebook:
+
+* Be sure that you have Python 3.9+ and Poetry
```

Comment:
In https://github.com/Ed-Fi-Alliance-OSS/LMS-Toolkit/blob/main/docs/demonstration/canvas-e2e.ipynb?short_path=68d8099#L13 and other notebooks, the instructions request for Python 3.9.x. I think these change suggests that you can use versions such as Python 3.10 with the LMS toolkit. Not sure if this is an actual requirement but seems like a mismatch.

Reply:
Good call, we haven't tested with 3.10 so no idea if there is some kind of conflict there.